fairlearn/datasets/_fetch_data.py

```diff
@@ -0,0 +1,91 @@
+import warnings
+from ._common import FairnessWarning
+
+from sklearn.datasets import fetch_openml
+
+
+def fetch_boston(*, version='1', cache=True, data_home=None,
+                 as_frame=False, return_X_y=False, warn=True):
+    """Load the boston housing dataset (regression).
+
+    Download it if necessary.
+
+    ==============   ==============
+    Samples total               506
+    Dimensionality               13
+    Features                   real
+    Target           real 5. - 50.
+    ==============   ==============
+
+    This dataset has known fairness issues. There's a "lower status of
+    population" (LSTAT) parameter that you need to look out for and a column
+    that is a derived from the proportion of people with a black skin color
+    that live in a neighborhood (B). This dataset should be used to explore
+    and benchmark fairness. Predicting a house-price while ignoring these
+    sensitive attributes is naive.
+
+    Parameters
+    ----------
+    version : integer or 'active', default='active'
+        Version of the dataset. Can only be provided if also ``name`` is given.
+        If 'active' the oldest version that's still active is used. Since
+        there may be more than one active version of a dataset, and those
+        versions may fundamentally be different from one another, setting an
+        exact version is highly recommended.
+
+    cache : boolean, default=True
+        Whether to cache downloaded datasets using joblib.
+
+    data_home : optional, default: None
+        Specify another download and cache folder for the datasets. By default
+        all scikit-learn data is stored in '~/scikit_learn_data' subfolders.
+
+    as_frame : boolean, default=False
+        If True, the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string or categorical). The target is
+        a pandas DataFrame or Series depending on the number of target_columns.
+        The Bunch will contain a ``frame`` attribute with the target and the
+        data. If ``return_X_y`` is True, then ``(data, target)`` will be pandas
+        DataFrames or Series as describe above.
+
+    return_X_y : boolean, default=False.
+        If True, returns ``(data.data, data.target)`` instead of a Bunch
+        object.
+
+    warn : boolean, default=False.
```

Comment:
I think the default value should be `True`, and the user can disable it explicitly.

Reply:
My bad! I agree.

Reply:
Fixed.